### DISCUSSION TITLE
RaSCSI 3.0 Ideas

### DISCUSSION THREAD

**Discussion by akuker** [akuker/RASCSI#701] [Ideas]
I'd like to start collecting feature ideas/improvements for a potential "RaSCSI 3.0". Some potential updates that I'm considering:
- Adding super capacitors and a minimal power management subsystem.  This would allow the pi to safely shut down when power is removed.
- Switching out transceivers to (potentially) either the SN74ALVTH162245 or the SN74F245DW
- Add a small microcontroller to help manage the rotary encoder on the microcontroller (and potentially help with power management)
- Switch to active termination or an actual SCSI termination IC, such as UCC5606.

**Comment by MrGasS** [comment]
While these are great ideas, isn't there a risk that RaSCSI could become more expensive?
What are the advantages of active termination instead of the passive one?

**Comment by mikerm19** [comment]
I second the supercap (battery?) option. I was actually just playing with this myself. I have a UPS pi hat, but it does something strange to the RaSCSI. If I put it on, it stops being able to boot from any SCSI device. If I take the hat back off again, it still won't boot until I run easyinstall and choose option 1 to rebuild it and reboot again.

**Comment by RolopoloRolo** [comment]
Would be a great feature to add some functionality to help clone physical devices. Done perfectly this would need the pi to be the  controlling host. I forget my scsi terminology. Very tricky stuff but perhaps some tools to aid in creating images of the right size. Sometimes getting all the physical space and cabling together to do it all on another system is a nightmare. Got a g3 laptop with scsi for that.

**Comment by jameswood** [comment]
Have you thought about basing it around the CM4?
